README.md

```diff
@@ -1,7 +1,247 @@
+| Local FDR                                                   | whether to compute a per-PSM/per-peptide pair etc. score based on a windowed FDR approach. Black box=compute local FDR, ticked box=calculate and filter by local FDR, white box=do not calculate local FDR                                         | to analyse distributions of scores and FDR or to generate a per-crosslink error estimate e.g. for structure modeling. |
+| min Pep. length                                             | minimum length of peptide 1 and peptide 2                                                                                                                                                                                                          | increase to 6 if noise matching of short sequences                                                                    |
+| min supporting peptide pairs Protein Group/Residue Pair/PPI | minimum number of peptide pairs necessary to aggregate to a given level                                                                                                                                                                            | define to have more robust network especially at the PPI level.                                                       |
+| min TD chance                                               | require enough target matches so that these number of decoy matches would still pass the target FDR. In other words, a minimum number for reliable FDR calculation                                                                                 | Use for more accurate FDR calculation. By default, this number is ignored because "ignore validity checks" is ticked  |
+| Ignore validity checks                                      | Allow FDR calculation even if number of decoys is not sufficient for sufficiently accurate estimation of FDR                                                                                                                                       | Untick and use in combination with min TD chance to only get results with accurate FDR calculation                    |
+| Ambiguity                                                   | increase number to only allow matches with at most N ambiguity. E.g. if "ambiguity" is set to 1 in the residue pairs, residue pairs that could come from more than 1 place in the sequence or more than 2 unique proteins are discarded            | Set to a number if not interested in considering ambiguous crosslinks at all |
+| Unique PSMs                                                 | Aggregate specral matches so that only the top scoring spectrum is considered per spectral match                                                                                                                                                   | on by default. Untick if interested in an analysis of spectral matches                                                                       |
```

Comment:
```suggestion
| Unique PSMs                                                 | Aggregate spectra matches so that only the top scoring spectrum match per peptide pair, charge state, and linkage sites combination is considered                                                                                                                                                    | on by default. Untick if interested in an analysis of spectral matches                                                                       |
```